base/ltexpl.dtx

```diff
@@ -516,4 +516,27 @@
 %<latexrelease>\EndIncludeInRelease
 %    \end{macrocode}
 %
+% \DescribeMacro\IfExplAtLeastTF
+%   A pretty simple wrapper.
+%
+% \begin{macro}{\IfExplAtLeastTF}
+% \changes{v1.3g}{2023/10/12}{Provide a test for \pkg{expl3} date (gh/1004)}
+%    \begin{macrocode}
+%<*2ekernel|latexrelease>
+%<latexrelease>\IncludeInRelease{2022/06/01}%
```

Comment:
Is the date 2022/06/01 specifically chosen? The next release should be 2023/11/01.